package.json

```diff
@@ -43,6 +43,7 @@
     "deploy": "webpack --config webpack.config -p && gh-pages -d build",
     "dependency-markdown": "node scripts/dependency-markdown-generator/DependencyMarkdownGenerator.js",
     "docs": "node markdown.config.js",
+    "docs:peerDependencies": "node scripts/peer-dependency-generator/index.js",
```

Comment:
Currently, this node script lives in terra-core, but we'lll want to use it across the monorepos. Curious what others thinking about a place to put this script show it can be shared across packages.

Reply:
We could either put it in toolkit or possibly make a new repo (this comes back to breaking up toolkit into a mono repo of smaller packages)

Reply:
Yeah, let's chat some more on this and figure out if we can find a good place for scripts like this.